src/views/components/trace/trace-detail-chart-table.vue

```diff
@@ -0,0 +1,148 @@
+<template>
```

Comment:
![image](https://user-images.githubusercontent.com/19775780/62281552-90395200-b480-11e9-9caf-c08c620fb2cd.png)

warning need fixed

Reply:
use <script lang='js'> not<script type="text/javascript"> to fix